lib/fastlane/core_ext/string.rb

```diff
@@ -0,0 +1,5 @@
+class String
+  def classify
+    split('_').collect!(&:capitalize).join
```

Comment:
Prefer `map!` over `collect!`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.